### DISCUSSION TITLE
Unable to Access GitHub Account

### DISCUSSION THREAD

**Discussion by RADiptho** [community/community#145998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I deleted the Gmail that was associated with my GitHub. Now how can I access my GitHub? My Facebook ID was linked to my GitHub.

**Comment by hassamohammed** [comment]
If you've deleted the Gmail account associated with your GitHub but your Facebook ID is linked, you can still access your GitHub account by following these steps:

Login with Facebook:

Go to the [GitHub login page](https://github.com/login).
Click "Sign in with Facebook."
Use your Facebook credentials to log in.
Reset your primary email:

Once you're logged in through Facebook, go to Settings > Emails and add a new active email address to your GitHub account.
Verify the new email address to make it your primary email.
Recover GitHub account (if Facebook fails):

If the Facebook login isn’t working, go to [GitHub Support](https://support.github.com/) and raise a ticket explaining the situation.
Provide details like:
Your GitHub username.
The email associated with the account (even if it’s deleted).
Proof of ownership (e.g., prior emails from GitHub or repository details).
Reactivate Gmail (if possible):
If the Gmail account was recently deleted, you may be able to recover it through Google Account Recovery (link here). Once recovered, you can use it to reset your GitHub credentials if needed.

Let me know if you need help with any of these steps!
